src/script/common/c_packer.h

```diff
@@ -71,6 +71,7 @@ struct PackedInstr
 		- userdata: name in registry
 	*/
 	std::string sdata;
+	std::string sdata2; // table: name of the metatable
```

Comment:
my thoughts on this: rather than adding an extra field that is unused 99% of times, I would add an `INSTR` for setting the metatable (and that can use `sdata`)